helpers/checkOfDraftOrFuturePost.ts

```diff
@@ -0,0 +1,19 @@
+import { PostDocumentWithoutContent } from "interfaces";
+
+const today = new Date();
+
+export function isPostADraft(post: PostDocumentWithoutContent) {
+  if (post.draft) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+export function isPostInTheFuture(post: { date: Date | string }) {
+  if (today.toISOString() < post.date.toString()) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
@ntorbinskiy did you test this function?

![image](https://user-images.githubusercontent.com/46556479/202861028-93b40bb7-ef4d-47c0-8900-87fbf92e7740.png)

Reply:
Function works properly. When you give yesterday-date to the function - you don't translate it to ISO string. That's why, it isn't working
![image](https://user-images.githubusercontent.com/60061661/202988503-5821d555-b9c4-4239-8646-910ad2465944.png)
![image](https://user-images.githubusercontent.com/60061661/202988571-efd986f2-7dcc-4d10-8307-d8da13e0b324.png)

Reply:
@ntorbinskiy 
So, you're comparing a string value with a number value, and you're saying it works correctly
?
![image](https://user-images.githubusercontent.com/46556479/203007203-d5c848ae-4c40-4e4b-b479-8c6b6395dc36.png)

Reply:
![image](https://user-images.githubusercontent.com/46556479/203007583-ca35d7db-c838-4244-8f80-243d982feb9d.png)

Reply:
This works because we use Date syntaxis. For example, "2017/10/26" > "2017/10/7" gives false because we use "/" instead of "-"

Reply:
![image](https://user-images.githubusercontent.com/60061661/203129944-dec1b33d-44bf-4f50-acec-75b91b843501.png)
https://stackoverflow.com/questions/14781153/how-to-compare-two-string-dates-in-javascript